pandas/io/excel.py

```diff
@@ -74,100 +168,13 @@ def get_writer(engine_name):
         raise ValueError("No Excel writer '%s'" % engine_name)
 
 
+@Appender(_read_excel_doc)
 def read_excel(io, sheetname=0, header=0, skiprows=None, skip_footer=0,
                index_col=None, names=None, parse_cols=None, parse_dates=False,
                date_parser=None, na_values=None, thousands=None,
                convert_float=True, has_index_names=None, converters=None,
                engine=None, squeeze=False, **kwds):
-    """
```

Comment:
Well, it's to be able to automatically inject the NA_VALUES instead of manually writing it in the docstring. 

It's a bit a trade-off here: is the added complexity of moving the docstring away from the actual function worth it for the 1 line of na values that is written manually (and has to be updated if the default recognized NaN values are changed).